app/templates/views/gallery_view.html

```diff
@@ -0,0 +1,44 @@
+{% extends "/snips/base.html" %}
+{% block title %}Submissions{% endblock %}
+{% block body %}
+
+<div class="inner">
+        <h1 style="text-align:center;">Submissions</h1>
+          <hr class="intro-divider">
+          <div class="center">
+            <table class="table table-striped table-bordered">
+   <thead>
+    <tr style="background-color: #004154; color: white;">
+       
+      <th width="10%">Name </th>
```

Comment:
Please use bootstrap grid system to instead of the width modifier
https://getbootstrap.com/docs/4.0/layout/grid/